content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```diff
@@ -31,7 +31,33 @@ $ java -jar GameLiftLocal.jar -p 9080
 
 ```
 
-## 2. Start Server
+## 2. Set `sv_useGameLiftServer` to `true`
+
+When you are ready to test with GameLift, opt-in by setting the `sv_useGameLiftServer` CVAR to `true`. This CVAR is off by default to facilitate local testing prior to the integration of the GameLift Server SDK into your game server.
```

Comment:
```suggestion
When you are ready to test with GameLift, opt-in by setting the `sv_useGameLiftServer` CVAR to `true`. This CVAR is off by default to facilitate local testing, prior to the integration of the GameLift Server SDK into your game server.
```

Reply:
this actually reads more intuitively to me w/o the comma